src/components/Preview.tsx

```diff
@@ -39,6 +39,17 @@ export class Preview extends React.Component<
     });
   }
 
+  componentWillReceiveProps(nextProps): void {
```

Comment:
This method will be deprecated on v17, I do not think any new code should use it.

https://reactjs.org/docs/react-component.html#the-component-lifecycle

Reply:
I created a WIP PR where I transform the Preview component into a functional component with a custom hook, but I need to properly test that before merging. I will merge this one at the moment and get back to that later

Reply:
My WIP PR is this: https://github.com/andrerpena/react-mde/pull/231